frontend/crocs-shop/src/app/app.component.ts

```diff
@@ -23,6 +24,20 @@ export class AppComponent {
+      name: 'Blue crocs',
+      price: '50$'
     }
   ]
+
+  generateRandomIntegerInRange(min: number, max: number) : number {
```

Comment:
Remove function, we will not use it. Also remove the properties that are involved in this function